apps/web/utils/constants.ts

```diff
@@ -11,3 +11,11 @@ if (!process.env.NEXT_PUBLIC_BACKEND_URL) {
 
 export const BASE_API_ENDPOINT = process.env.NEXT_PUBLIC_BACKEND_URL;
 export const COURSE_API_ENDPOINT = `${BASE_API_ENDPOINT}/courses`;
+
+export const APPROVED = "APPROVED";
+export const REJECTED = "REJECTED";
+export const PENDING = "PENDING";
```

Comment:
Great idea actually, I'll use it on my page as well. 

Though maybe we can give it a little more context? Like:
- `REVIEW_STATUS_APPROVED`
- `REVIEW_STATUS_PENDING`
- `REVIEW_STATUS_REJECTED`